app/api/ada/lib/storage/database/uncategorized/api/get.js

```diff
@@ -0,0 +1,116 @@
+// @flow
+
+import type {
+  lf$Database,
+  lf$Transaction,
+} from 'lovefield';
+
+import type {
+  ConceptualWalletRow,
+  KeyRow,
+} from '../tables';
+import * as Tables from '../tables';
+import { getRowFromKey } from '../../utils';
+import { Bip44WrapperSchema, } from '../../genericBip44/tables';
+import type { Bip44WrapperRow } from '../../genericBip44/tables';
+
+export class GetConceptualWallet {
+  static ownTables = Object.freeze({
+    [Tables.ConceptualWalletSchema.name]: Tables.ConceptualWalletSchema,
+  });
+  static depTables = Object.freeze({});
+
+  static async get(
+    db: lf$Database,
+    tx: lf$Transaction,
+    key: number,
+  ): Promise<ConceptualWalletRow | void> {
+    const walletSchema = GetConceptualWallet.ownTables[Tables.ConceptualWalletSchema.name];
+    return await getRowFromKey<ConceptualWalletRow>(
+      db, tx,
+      key,
+      walletSchema.name,
+      walletSchema.properties.ConceptualWalletId,
+    );
+  }
+}
+
+export class GetKey {
+  static ownTables = Object.freeze({
+    [Tables.KeySchema.name]: Tables.KeySchema,
+  });
+  static depTables = Object.freeze({});
+
+  static async get(
+    db: lf$Database,
+    tx: lf$Transaction,
+    key: number,
+  ): Promise<KeyRow | void> {
+    return await getRowFromKey<KeyRow>(
+      db, tx,
+      key,
+      GetKey.ownTables[Tables.KeySchema.name].name,
+      GetKey.ownTables[Tables.KeySchema.name].properties.KeyId,
+    );
+  }
+}
+
+const WalletType = Object.freeze({
+  Bip44: 'bip44',
```

Comment:
I don’t know if it’s important to define, but maybe we should follow some sort of cap standard.

Reply:
not sure if I understand what you mean